medusa/server/web/home/handler.py

```diff
@@ -956,8 +956,6 @@ def massEditShow(
             errors += 1
             return errors
 
-        series_obj.aliases = get_scene_exceptions(series_obj)
```

Comment:
@medariox any idea why this assignment is here?

Reply:
No idea

Reply:
well it's gone now, prob fixed